components/fxa-client/src/internal/oauth.rs

```diff
@@ -38,18 +38,20 @@ impl FirefoxAccount {
     /// using `begin_oauth_flow`.
     ///
     /// * `scopes` - Space-separated list of requested scopes.
-    /// * `ttl` - the ttl in seconds of the token requested from the server.
+    /// * `use_cache` - optionally set to false to force a new token request.
```

Comment:
maybe note that the returned token is cached even when `use_cache = false`? (if you do, I guess in the udl and other places this is copied too?)